source/frontend/src/features/mapSideBar/router/FilePropertyRouter.tsx

```diff
@@ -156,7 +156,11 @@ export const FilePropertyRouter: React.FC<IFilePropertyRouterProps> = props => {
 const getFileProperty = (file: ApiGen_Concepts_File, selectedFilePropertyId: number) => {
   const properties = file?.fileProperties || [];
 
-  return properties.find(p => p.id === selectedFilePropertyId);
+  const fileProperty = properties.find(p => p.id === selectedFilePropertyId);
+  if (exists(fileProperty) && !exists(fileProperty.file)) {
+    fileProperty.file = file;
+  }
+  return fileProperty;
```

Comment:
this was addressed in another PR - I guess when we merge that one this change will go away